keys.go

```diff
@@ -130,24 +137,48 @@ func (c *Client) CreateRootKey(ctx context.Context, name string, expiration *tim
 	return c.CreateKey(ctx, name, expiration, false)
 }
 
+// CreateRootKeyWithSHA1 creates a new, non-extractable key resource without
+// key material using RSAES OAEP SHA 1 as encryption algorithm.
+func (c *Client) CreateRootKeyWithSHA1(ctx context.Context, name string, expiration *time.Time) (*Key, error) {
+	return c.CreateKeyWithSHA1(ctx, name, expiration, false)
+}
+
```

Comment:
There is nothing to import here so this doesn't make sense either.

Reply:
Done.